test/e2e/cmd/run/run.go

```diff
@@ -297,29 +297,71 @@ func (h *helper) createRoles() error {
 	return h.kubectlApplyTemplateWithCleanup("config/e2e/roles.yaml", h.testContext)
 }
 
-func (h *helper) installOperator() error {
-	log.Info("Installing the operator")
-	values, err := h.renderTemplate("config/e2e/helm-values.yaml", h.testContext)
-	if err != nil {
-		return fmt.Errorf("failed to generate Helm values: %w", err)
+func (h *helper) installE2EOperator() error {
```

Comment:
Should this be called something like `operatorUnderTest` rather then e2e operator (also in other places). Just to make clear it is the test subject?